app-misc/ola/ola-0.10.7.ebuild

```diff
@@ -0,0 +1,100 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python2_7 python3_{6,7} )
+
+inherit autotools python-single-r1
+
+if [[ "${PV}" == "9999" ]]; then
+	inherit git-r3
+
+	EGIT_REPO_URI="https://github.com/OpenLightingProject/${PN}"
+fi
+
+DESCRIPTION="Open Lighting Architecture"
+HOMEPAGE="http://www.openlighting.org/"
```

Comment:
https://www.openlighting.org/

Reply:
Very true, thanks for noticing. I changed it.